app/src/main/resources/spring-config.xml

```diff
@@ -58,7 +58,7 @@
 	</bean>
 
 	<bean id="logger" class="com.oms.util.Logger">
-		<property name="path" value="oms-log.txt"/>
+		<property name="path" value="/var/log/oms/oms-log.txt"/>
```

Comment:
Perhaps it is a good idea to put the logs under /var/log but it was not really required in the issue.